libs/webb-ui-components/src/components/ConnectWalletMobileButton/ConnectWalletMobileButton.tsx

```diff
@@ -38,24 +45,44 @@ export const ConnectWalletMobileButton = forwardRef<
         </Button>
       </BottomDialogTrigger>
       <BottomDialogPortal
-        title="Try Hubble on Desktop"
-        actionButtonsProps={actionButtonsProps}
+        title={
+          appType === 'bridge-dapp'
+            ? 'Try Hubble on Desktop'
+            : 'Switch to Desktop'
+        }
+        actionButtonsProps={
+          appType === 'bridge-dapp'
+            ? bridgeActionButtonsProps
+            : tangleActionButtonsProps
+        }
         className="w-full"
       >
         <div className="flex flex-col gap-4">
-          <Typography variant="body1">
-            A complete mobile experience for Hubble Bridge is in the works. For
-            now, enjoy all features on a desktop device.
-          </Typography>
-          <Typography variant="body1">
-            Visit the link on desktop below to start transacting privately!
-          </Typography>
-          <Typography
-            variant="body1"
-            className="text-[#3D7BCE] dark:text-[#81B3F6]"
-          >
-            {BRIDGE_URL}
-          </Typography>
+          {appType === 'bridge-dapp' ? (
+            <>
+              <Typography variant="body1">
+                A complete mobile experience for Hubble Bridge is in the works.
+                For now, enjoy all features on a desktop device.
+              </Typography>
+              <Typography variant="body1">
+                Visit the link on desktop below to start transacting privately!
+              </Typography>
+              <Typography
+                variant="body1"
+                className="text-[#3D7BCE] dark:text-[#81B3F6]"
+              >
+                {BRIDGE_URL}
+              </Typography>
+            </>
+          ) : (
+            <div className="flex flex-col justify-center items-center gap-4 py-9">
+              <ComputerIcon size="xl" className="mx-auto" />
+              <Typography variant="body1" className="text-center">
+                For the best staking experience, we recommend using our desktop
+                interface for full-feature interface and enhanced controls.
+              </Typography>
+            </div>
+          )}
```

Comment:
For this, utilizing the `children` prop and use the existing component as the default.